_resources/port1.0/group/python-1.0.tcl

```diff
@@ -325,46 +315,22 @@ proc python_get_defaults {var} {
             }
         }
         lib {
-            if {${python.version} != 24 && ${python.version} != 25} {
-                return "${python.prefix}/Python"
-            } else {
-                return "${prefix}/lib/libpython${python.branch}.dylib"
-            }
+            return "${prefix}/lib/libpython${python.branch}.dylib"
         }
         pkgd {
-            if {${python.version} != 24} {
-                return "${python.prefix}/lib/python${python.branch}/site-packages"
-            } else {
-                return "${prefix}/lib/python${python.branch}/site-packages"
-            }
+            return "${prefix}/lib/python${python.branch}/site-packages"
         }
         setup_args {
-            if {${python.version} != 24} {
-                return "--no-user-cfg"
-            } else {
-                return ""
-            }
+            return "--no-user-cfg"
```

Comment:
All these values don't actually need to be using python_get_defaults if there's no longer any decision-making happening.

Reply:
could you please expand on your comment ... I don't get the meaning :-)